bzt/engine.py

```diff
@@ -671,6 +690,20 @@ def masq_sensitive(value, key, container):
                     if value and isinstance(value, (string_types, text_type)):
                         container[key] = '*' * 8
 
+    def _replace_tabs(self, lines, fname):
+        has_tab_intents = re.compile("^( *)(\t+)( *\S*)")
+        res = ""
+        for num, line in enumerate(lines):
+            replaced = has_tab_intents.sub(r"\1" + (" " * self.tab_replacement_spaces) + r"\3", line)
+            if replaced != line:
+                line = replaced
+                if self.warn_on_tab_replacement:
+                    self.log.warning("Replaced leading tabs in file %s, line %s", fname, num)
+                    self.log.warning("Line content is: %s", replaced.strip())
+                    self.log.warning("Please remember that YAML spec does not allow using tabs for indentation")
```

Comment:
I won't suggest that half-working option :) Let's see how users will face this "feature" first.